01-Alex-Nicolaidis/wk01/5-fri/mta/js/mta.js

```diff
@@ -0,0 +1,77 @@
+//MTA
+
+//stations array for each line
+const lineN = [ 'Times Square', '34th', '28th', '23rd', 'Union Square', '8th' ];
+const lineL = [ '8th', '6th', 'Union Square', '3rd', '1st' ];
+const line6 = [ 'Grand Central', '33rd', '28th', '23rd', 'Union Square', 'Astor Place'];
+
+//create a factory to generate the train lines and station
+
+const trainFactory = function (l,s){
+    return {
+        line: l,
+        stations:s
+    }
+};
+
+const mtaTrains = [trainFactory("N",lineN), trainFactory("L",lineL), trainFactory("6",line6)];
```

Comment:
If you structure your `mtaTrains` variable as an object instead of an array then you woudn't need your `getStations()` function. See [this comment](https://github.com/Tenzang/SEIC57-homework/pull/33#discussion_r986281419) on Michael's PR for how that could be implemented.